src/dso_api/dynamic_api/views/mvt.py

```diff
@@ -152,6 +140,37 @@ def get(self, request, *args, **kwargs):
         )
         return result
 
+    def _schemafy_layer(self, layer: BaseVectorLayerMixin) -> None:
+        schema: DatasetTableSchema = self.model.table_schema()
+        user_scopes: UserScopes = self.request.user_scopes
+
+        layer.geom_field = schema.main_geometry_field.python_name
+
+        queryset = self.model.objects.all()
+        tile_fields = ()
+
+        for field in schema.fields:
+            field_name = field.name
+            if not user_scopes.has_field_access(field):
+                # 403
+                continue
+            if field.is_relation:
+                # Here we have to use the db_name, because that usually has a suffix not
+                # available on field.name.
+                field_name = toCamelCase(field.db_name)
+            if field_name != field.db_name and any(char.isupper() for char in field_name):
```

Comment:
```suggestion
            if field_name != field.db_name and field_name.lower() != field_name:
```

Misschien net zo makkelijk?

En en tegenstelling wat je zou verwachten nog sneller ook:

```
>>> timeit(lambda: any(char.isupper() for char in field_name))
0.8461196800053585
>>> timeit(lambda: field_name.lower() != field_name)
0.09950922799180262
```